client/components/RatingsReviews/ratingCalculations.jsx

```diff
@@ -0,0 +1,19 @@
+
+const ratingCalculations = function (allRatings) {
+  let total = 0;
+  let ratingsCount = 0;
+  for (let x = 1; x < 6; x++) {
+    if (allRatings[x]) {
+      total += allRatings[x] * x;
+      ratingsCount += parseInt(allRatings[x]);
```

Comment:
no need to parseInt here. as demonstrated in the line above, entries in the allRatings object are already numbers. If there is some precision weirdness, consider using Math.floor instead or something.

Reply:
Weirdly enough, `allRatings[x]` is actually a string, and multiplication for some reason works on strings if the string is a number. I had to double check, but for example, `'3' * 2 === 6`.

Reply:
weird af, disregard my comment then